src/Module.php

```diff
@@ -190,6 +190,7 @@ protected function registerTranslation()
 
         if (is_dir($langPath)) {
             $this->loadTranslationsFrom($langPath, $lowerName);
+            $this->loadJSONTranslationsFrom($langPath);
```

Comment:
This should be `loadJsonTranslationsFrom`. JSON is not capitalized: [Illuminate/Support/ServiceProvider.php#L107-L116](https://github.com/laravel/framework/blob/dfcb8e372e7efb0e95c5844571c0701e91b7f2e6/src/Illuminate/Support/ServiceProvider.php#L107-L116)